client/app/src/components/account/templates/account-card.html

```diff
@@ -64,9 +64,15 @@
       </md-icon>
       </span>
       <span class="md-subhead description">
-          <translate class="balance">Balance</translate> {{$ctrl.ul.network.symbol}} {{$ctrl.ul.selected.balance/100000000}}
-          <md-button ng-click="$ctrl.ul.selectNextCurrency()">{{$ctrl.ul.selected.balance/100000000 | amountToCurrency:$ctrl | formatCurrency:$ctrl.ul }}</md-button>
-        </span>
+        <translate class="balance">Balance</translate> {{$ctrl.ul.network.symbol}} {{$ctrl.ul.selected.balance/100000000}}
```

Comment:
It'd be nice to standardize the 100000000 value as a variable. @j-a-m-l created a 'UNIT' variable (for arkthosi's) in one of his recent commits I believe. Not sure the variable is exposed to this logic here though.

I don't think it's necessary for it to be apart of this pull request, but something to keep in mind if we see it keep coming up.

Reply:
Good point @spresnal - I'll raise a git issue as you're right, not sure it relates to the toggle, but it definitely needs updating for consistency. Thanks!

Reply:
Yes, @spresnal I created that `UNIT` constant on different places as a transition to centralized its value as a configuration.

BTW: do we have a better name than "arkthosi"?

Reply:
I've raised this as a separate PR - #404 (not found? 😂)